packages/build/src/core/commands.js

```diff
@@ -35,7 +35,7 @@ const getEventCommands = function({
     return pluginCommands
   }
 
-  const shellCommandA = { id: `config.build.lifecycle.${event}`, event, shellCommand, override: {} }
```

Comment:
Why does this remove the `override` as well? I thought that was implemented in https://github.com/netlify/build/pull/979/files#diff-d0ced5cdf67a34d175b13ec316d7a7f9L38.

Reply:
This PR needs some changes from #979. Therefore it includes the diff from that other PR at the moment. When #979 is merged, I will rebase this PR so the diff does include that other PR anymore.

Reply:
This is rebased now. :+1: